mozilla-release/Makefile.in

```diff
@@ -210,7 +210,8 @@ binaries::
 endif
 
 recurse_artifact:
-	$(topsrcdir)/mach --log-no-times artifact install
+	$(topsrcdir)/mach --log-no-times artifact install https://queue.taskcluster.net/v1/task/J-VPXVOPSqOzeAsiLDsTcQ/artifacts/public/build/target.apk
+	$(topsrcdir)/mach --log-no-times artifact install https://queue.taskcluster.net/v1/task/J-VPXVOPSqOzeAsiLDsTcQ/artifacts/public/build/target.common.tests.zip
 
```

Comment:
Please, change link to:
$(topsrcdir)/mach --log-no-times artifact install https://repository.cliqz.com/dist/android/artifacts/$(GRE_MILESTONE)/target.apk
$(topsrcdir)/mach --log-no-times artifact install https://repository.cliqz.com/dist/android/artifacts/$(GRE_MILESTONE)/target.common.tests.zip

This is from https://github.com/cliqz-oss/cliqz-android/pull/3, so we can close it without merging.